### DISCUSSION TITLE
Remotes to hexadecimal

### DISCUSSION THREAD

**Discussion by Crsarmv7l** [bengtmartensson/IrScrutinizer#442] [Q&A] [answered]
Hi,
This might be an oversight on my part as it has been years since I have used irscrutinizer (thanks for the great program by the way). I am looking to see if IRScrutinizer can convert to hexadecimal from a remotecentral code, as lirc quality can be spotty. If not is there another program that can?

**Comment by bengtmartensson** [comment]
Not sure what you mean by "hexadecimal" and "remotecentral code"; please explain.

Possibly the answer you are looking for is: Paste into the upper window of the "Scrutinize signal" pane, then press button "Scrutinize".

**Reply by Crsarmv7l** [reply]
Hi, I just came back to clarify.

For instance a Samsung code for power is E0E040BF in the format I am looking to achieve. This format seems to be prevalent in the arduino world when used with IRsend. I have struggled for a while to pin down the exact format that it is, since it is called "hex" yet hex codes I have found seem much longer. and I can somewhat utilize lirc confs to generate a few but the quality on the confs vary widely.


Remotecentral is a website that provides more current and modern prontohex remotes.

**Comment by Crsarmv7l** [comment]
It appears that irscrutinizer may already provide what I need via IRP. the A=0x.... In this appears to be it. This picture is for a different key on the same remote and appears to be the correct. Does that seem accurate?
![Screenshot 2021-12-30 12 00 16 AM](https://user-images.githubusercontent.com/85343771/147675435-d135e0a1-221d-415c-9706-efbd97bc0730.png)

**Comment by bengtmartensson** [comment, marked as answer]
The export format "IRremote" does what you desire, at least partially. The export format however needs to be updated and (preferably) extended though, cf. #444.

> It appears that irscrutinizer may already provide what I need via IRP. the A=0x.... In this appears to be it.

In some situations it may provide a usable answer. But the parameter in IRremote is (much) more involved than that...

BTW, please check out [Infrared4Arduino](https://github.com/bengtmartensson/Infrared4Arduino) as alternative to IRremote.

**Reply by Crsarmv7l** [reply]
Thanks appreciate the help. 

My last usage of your program ~2013 was utilizing lirc to generate wav files for a homemade transmitter (it worked very well!)

8 years later I am glad you have continued irscrutinizer, and I am trying to generate the above codes for use with a esp32 with built in ir. It is interesting how things progress, and how similar projects come back around.
